esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3567,6 +3596,78 @@ bool CFileSprayEx::onDeleteDropZoneFiles(IEspContext &context, IEspDeleteDropZon
     return true;
 }
 
+void CFileSprayEx::checkDropZoneFileScopeAccess(IEspContext &context, const char *dropZoneName, const char *netAddress,
+    const char *dropZonePath, const StringArray &dropZoneFiles, SecAccessFlags accessReq)
+{
+    const char *accessReqName = getSecAccessFlagName(accessReq);
+    const char *dropZoneId = isContainerized() ? dropZoneName : netAddress;
```

Comment:
see prev. comments, I think this and other places should unify on dropZoneName only.